src/components/file/examples/file-default.tsx

```diff
@@ -21,9 +24,9 @@ export class FileExample {
         return [
             <limel-file
                 label="Attach a file"
-                value={this.value}
-                required={this.required}
                 onChange={this.handleChange}
+                required={this.required}
+                value={this.value}
```

Comment:
This is another change that doesn't belong in this commit. I'll fix it for you this time, but for next time, if you absolutely have to sort stuff, you know to do it beforehand 🙂

Reply:
🙏